chain/cmd/bandoracled2/executor.go

```diff
@@ -10,20 +12,41 @@ type executor interface {
 	Execute(l *Logger, exec []byte, timeout time.Duration, arg string) ([]byte, uint32)
 }
 
-type lambdaExecutor struct{}
+type lambdaExecutor struct {
+	URL string
+}
 
 func (e *lambdaExecutor) Execute(
 	l *Logger, exec []byte, timeout time.Duration, arg string,
 ) ([]byte, uint32) {
-	// TODO: Make URL configurable
-	result, err := byteexec.RunOnAWSLambda(
-		exec, timeout, arg,
-		"https://dmptasv4j8.execute-api.ap-southeast-1.amazonaws.com/bash-execute",
-	)
+	result, err := byteexec.RunOnAWSLambda(exec, timeout, arg, e.URL)
 	if err != nil {
 		l.Error(":skull: LambdaExecutor failed with error: %s", err.Error())
 		return []byte("EXECUTION_ERROR"), 255
 	}
 
 	return result, 0
 }
+
+// NewExecutor returns executor by name and executer URL
+func NewExecutor(executer string) (executor, error) {
+	name, url, err := ParseExecutor(executer)
+	if err != nil {
+		return nil, err
+	}
+	switch name {
+	case "lambda":
+		return &lambdaExecutor{URL: url}, nil
+	default:
+		return nil, fmt.Errorf("Invalid executor name: %s, url: %s", name, url)
+	}
+}
+
+// ParseExecutor returns parsed executor string
+func ParseExecutor(executorStr string) (name string, url string, err error) {
```

Comment:
This function can now be private (lowercase) now that it's not intended to be used outside.